apitools/base/py/encoding.py

```diff
@@ -328,12 +328,15 @@ def encode_message(self, message):
         if isinstance(message, messages.FieldList):
             return '[%s]' % (', '.join(self.encode_message(x)
                                        for x in message))
-        message_type = type(message)
-        if message_type in _CUSTOM_MESSAGE_CODECS:
+
+        # pylint: disable=unidiomatic-typecheck
```

Comment:
small nit: this actually disables the warning for the rest of this block; if you put it at the end of the line, it'll only disable it on that line.

(happily, the linter should be OK with a long line that's due to a pylint disable.)